add-ons/che/mini-che-template.yml

```diff
@@ -0,0 +1,380 @@
+                  valueFrom:
+                    configMapKeyRef:
+                      key: che-oauth-github-forceactivation
+                      name: che
+                - name: CHE_OPENSHIFT_WORKSPACE_MEMORY_OVERRIDE
+                  valueFrom:
+                    configMapKeyRef:
+                      key: workspaces-memory-limit
+                      name: che
+                - name: CHE_OPENSHIFT_WORKSPACE_MEMORY_REQUEST
+                  valueFrom:
+                    configMapKeyRef:
+                      key: workspaces-memory-request
+                      name: che
+                - name: CHE_WORKSPACE_AUTO__START
+                  valueFrom:
+                    configMapKeyRef:
+                      key: enable-workspaces-autostart
+                      name: che
+                - name: JAVA_OPTS
+                  valueFrom:
+                    configMapKeyRef:
+                      key: che-server-java-opts
+                      name: che
+                - name: CHE_WORKSPACE_JAVA_OPTIONS
+                  valueFrom:
+                    configMapKeyRef:
+                      key: che-workspaces-java-opts
+                      name: che
+                - name: CHE_OPENSHIFT_SECURE_ROUTES
+                  valueFrom:
+                    configMapKeyRef:
+                      key: che-openshift-secure-routes
+                      name: che
+                - name: CHE_DOCKER_SERVER__EVALUATION__STRATEGY_SECURE_EXTERNAL_URLS
+                  valueFrom:
+                    configMapKeyRef:
+                      key: che-secure-external-urls
+                      name: che
+                - name: CHE_OPENSHIFT_SERVER_INACTIVE_STOP_TIMEOUT_MS
+                  valueFrom:
+                    configMapKeyRef:
+                      key: che-server-timeout-ms
+                      name: che
+                - name: CHE_OPENSHIFT_PRECREATE_WORKSPACE_DIRS
+                  valueFrom:
+                    configMapKeyRef:
+                      key: che-openshift-precreate-subpaths
+                      name: che
+                - name: CHE_KEYCLOAK_OSO_ENDPOINT
+                  valueFrom:
+                    configMapKeyRef:
+                      key: keycloak-oso-endpoint
+                      name: che
+                - name: CHE_KEYCLOAK_GITHUB_ENDPOINT
+                  valueFrom:
+                    configMapKeyRef:
+                      key: keycloak-github-endpoint
+                      name: che
+                - name: CHE_KEYCLOAK_DISABLED
+                  valueFrom:
+                    configMapKeyRef:
+                      key: keycloak-disabled
+                      name: che
+                - name: CHE_WORKSPACE_AUTO__SNAPSHOT
+                  valueFrom:
+                    configMapKeyRef:
+                      key: che-workspace-auto-snapshot
+                      name: che
+              image: 'docker.io/eclipse/che-server:nightly'
```

Comment:
this should be from Image streams rather than particular image to handle image and tags betetr